src/turnin.c

```diff
@@ -208,10 +208,18 @@ void check_assignment(char* s){
 
 char * timestamp(time_t clock) {
 	char *b = (char *) malloc(16);
-	struct tm *t = localtime(&clock);
+	struct tm t;
+	struct tm *res = localtime_r(&clock, &t);
```

Comment:
I found this in the `man` page:

> According to POSIX.1-2004, localtime() is required to behave as though [tzset](https://linux.die.net/man/3/tzset)(3) was called, while localtime_r() does not have this requirement. For portable code [tzset](https://linux.die.net/man/3/tzset)(3) should be called before localtime_r().

Can you add this call please?

Reply:
Will do, thanks for the pointer.